src/body.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from 'react';
+import { getNasaImg } from './nasapic'
+import { StyledImg } from './StyledImg'
+import { StyledBody } from './StyledBody'
+
+const Body = () => {
+    const [img, setImg] = useState();
+    const [date, setDate] = useState();
+    const [des, setDes] = useState();
+
+    useEffect(() =>{
```

Comment:
I think in lecture Jason went over that it's better practice to keep the useEffect in tact meaning not extracting the logic, so it's easier to read similar to styled components